skimage/filters/_frangi.py

```diff
@@ -0,0 +1,179 @@
+import numpy as np
+
+__all__ = ['frangi', 'hessian']
+
+
+def _frangi_hessian_common_filter(image, scale_range, scale_step,
+                                  beta1, beta2):
+    """This is an intermediate function for Frangi and Hessian filters.
+
+    Shares the common code for Frangi and Hessian functions.
+
+    Parameters
+    ----------
+    image : (N, M) ndarray
+        Array with input image data.
+    scale_range : tuple of floats, optional
```

Comment:
Mmm... For me it seems like a tuple of any length is supported. Maybe put it explicit: `2-tuple of floats`?